plugins/ruby-foreman-azure-rm/debian/changelog

```diff
@@ -0,0 +1,5 @@
+ruby-foreman-azure-rm (2.2.0) unstable; urgency=medium
```

Comment:
```suggestion
ruby-foreman-azure-rm (2.2.0-1) unstable; urgency=medium
```

We've been using 3.0(native) and yet with debian revisions, to indicate changes to packaging.
Would you mind conforming tot that? :)